src/BaGet.AWS/Configuration/S3StorageOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace BaGet.AWS.Configuration
+{
+    public class S3StorageOptions
+    {
+        public string KeyId { get; set; }
+        public string KeySecret { get; set; }
+        public string Region { get; set; }
+        public string Bucket { get; set; }
+        public string Prefix { get; set; }
```

Comment:
Out of curiosity, what's the benefit of using a prefix instead of a new bucket? Over on Azure we make new containers (the equivalent of buckets) all the time.

Reply:
When downloading from the S3 bucket directly, its name dictates the URL. To allow more control of this, I prefer using prefixes - since it allows control of the latter part of the URL. 

I can remove it to be consistent?

Reply:
You can keep it in, I can see how it would be useful

Reply:
Since the annotation (and there was just one) isn't there, I've removed the reference.